tests/integration/it_table_shapes_test.py

```diff
@@ -34,16 +34,40 @@ def test_create(client, db_session):
     assert models.TableShape.query.count() == 1
 
 
+# @todo #206:30min Fix timeless.forms.ModelForm so it will use instance
+#  when populating fields in form, not only during save and update. After the
+#  it has been fixed enable two assertions below.
```

Comment:
@krzyk I don't really understand, what is the problem with ``ModelForm``? :D

Reply:
@amihaiemil problem is that the assertions should work but they don't :)
ModelForm doesn't update itself from the `instance` parameter, but it should (as it does when calling e.g. `save` but it doesn't do that when doing `form.description` or `form.picture`.